src/WrapperApp/components/Simulation/SimulationPanel.tsx

```diff
@@ -89,7 +89,7 @@ export default function SimulationPanel({
 				handleLoadResults={handleLoadResults}
 				handleShowInputFiles={handleShowInputFiles}
 			/>
-			{demoMode ? (
+			{/* {demoMode ? (
```

Comment:
why is this commented out ?

Reply:
I wanted to ask about this code on the meeting. I haven't analyzed it yet.

Reply:
The reason is simple, although any comments are missing. 


Lets consider frontend deployed as "demo version", for example on https://yaptide.github.io/web_dev/.
To deploy demo version you use `npm run start-demo` as described in main readme:
![image](https://github.com/yaptide/ui/assets/7374733/8666b2bf-e482-4d28-97a4-94df689dbacb)

In demo version "login" option is disabled. When you go to simulation tab in left drawer you will see "Demo simulation results" . Check https://yaptide.github.io/web_dev/ :
![image](https://github.com/yaptide/ui/assets/7374733/5b65283c-7229-48aa-aa66-02f60bfe474e)

In normal mode (`npm start-plg` or `npm run start`) when login option is enabled, if you go to simulation tab in left drawer you should see only "Local simulation results".